doc/api/fs.md

```diff
@@ -4778,8 +4802,8 @@ changes:
     represents the number of retries. This option is ignored if the `recursive`
     option is not `true`. **Default:** `0`.
   * `recursive` {boolean} If `true`, perform a recursive directory removal. In
-    recursive mode, errors are not reported if `path` does not exist, and
-    operations are retried on failure. **Default:** `false`.
+    recursive mode, operations are retried on failure. **Default:** `false`.
+    **Deprecated**.
```

Comment:
It's a bit hard to read this diff. Does this say the `recursive` option is deprecated? That shouldn't be the case for `rm` but I might be reading this wrong.

Reply:
No, it says that `recursive` option of `fs.rmDirSync` is deprecated. `fs.rm` is actually 1000 lines above if you check the line number, but I agree the diff is kinda hard to read here.